website_crm_quick_answer/data/base_automation_data.xml

```diff
@@ -22,22 +22,15 @@
                     <p>Please, be patient. Best regards.</p>
                     <p>Thanks,</p>
                 </div>
-            </field>
-        </record>
-
-    </data>
-
-    <data>
-
-        <record id="automated_action" model="base.automation">
-            <field name="name">Quick response to website contact form</field>
-            <field name="model_id" ref="crm.model_crm_lead" />
-            <field name="state">email</field>
-            <field name="trigger">on_create</field>
-            <field name="active" eval="True" />
-            <field name="template_id" ref="email_template" />
-            <field name="filter_domain">[["medium_id.name","=","Website"]]</field>
-        </record>
-
-    </data>
+        </field>
+    </record>
+    <record id="automated_action" model="base.automation">
+        <field name="name">Quick response to website contact form</field>
+        <field name="model_id" ref="crm.model_crm_lead" />
+        <field name="state">email</field>
+        <field name="trigger">on_create</field>
+        <field name="active" eval="True" />
+        <field name="template_id" ref="email_template" />
+        <field name="filter_domain">[["medium_id.name","=","Website"]]</field>
```

Comment:
This doesn't serve on translated terms. Have you tried to put `ref()` here to see if it works?